account_due_list/views/payment_view.xml

```diff
@@ -41,6 +40,7 @@
                 />
                 <field name="debit" readonly="1" sum="Total Debit" optional="hide" />
                 <field name="credit" readonly="1" sum="Total Credit" optional="hide" />
+                <field name="company_currency_id" invisible="1" readonly="1" />
```

Comment:
Why do you need to add this?

Reply:
Because without the company currency the sum totals fields cannot be displayed.

Without the company_currency_id:
![image](https://github.com/OCA/account-payment/assets/162969472/acaddfe0-0380-41ec-9aae-203a0e0918a1)


With the company_currency_id:
![image](https://github.com/OCA/account-payment/assets/162969472/d55fa9dd-740f-45cd-a318-9c1064d4af78)

Reply:
Thanks for the explanation. Then put a comment above the line indicating that. If not, someone (or your future you) will be tempted to remove the line thinking it's useless.